modules/flowable-engine/src/test/java/org/flowable/engine/test/bpmn/event/end/TerminateEndEventTest.java

```diff
@@ -1060,6 +1072,50 @@ public void testThreeExecutionsArrivingInTerminateEndEvent() {
         assertHistoricProcessInstanceDetails(processInstanceId);
     }
 
+    @Test
+    @Deployment
+    void testWithSubCaseTaskAndEnsureSubCaseIsTerminated() {
```

Comment:
Please don't use mockito. We want to test that the business logic is correct, not that our implementation does the proper calls. If this test needs the cmmn-engine then move it in the cmmn-engine-configurator, where we are testing the integration between the engines

Reply:
As you might have seen, this test also exists as an integration version within the `cmmn-engine-configurator`. I would also expect a test here, since this is part where the behavior of the terminate end event is tested, assuming it has an external `CaseInstanceService` which is doing what it is supposed to do. If you think this part is not necessary, I am happy to remove the test.

Reply:
This test is gone, since I moved the implementation to a different location